vips.go

```diff
@@ -503,15 +503,18 @@ func vipsSmartCrop(image *C.VipsImage, width, height int) (*C.VipsImage, error)
 	return buf, nil
 }
 
-func vipsTrim(image *C.VipsImage) (int, int, int, int, error) {
+func vipsTrim(image *C.VipsImage, background Color, threshold float32) (int, int, int, int, error) {
 	defer C.g_object_unref(C.gpointer(image))
 
 	top := C.int(0)
 	left := C.int(0)
 	width := C.int(0)
 	height := C.int(0)
 
-	err := C.vips_find_trim_bridge(image, &top, &left, &width, &height)
+	err := C.vips_find_trim_bridge(image,
+		&top, &left, &width, &height,
+		C.double(background.R), C.double(background.G), C.double(background.B),
+		C.double(threshold))
```

Comment:
`float32` and `C.double` ?

Reply:
Oh no! You're right. Should be float64 here, it's used as a double in libvips.